AWARDS.md

```diff
@@ -1,27 +1,27 @@
 # Awards
 
-|     | Icon | Name                 | Description                                         | Sentiment | Notes                                            |
-| --- | ---- | -------------------- | --------------------------------------------------- | --------- | ------------------------------------------------ |
+|     | Icon | Name                 | Description                                         | Sentiment | Notes                                       |
+| --- | ---- | -------------------- | --------------------------------------------------- | --------- | ------------------------------------------- |
 | ✅  |      | Kenny                | Most Deaths                                         | Negative  |
 | ✅  |      | Buddhist             | Lowest damage to champions                          | Negative  |
 | ✅  |      | Tree Hugger          | Lowest siege damage                                 | Negative  |
-| ✅  |      | Thrift Shopper       | Lowest value full build                             | Negative  | at least 2 teammates must have a full build      |
+| ✅  |      | Thrift Shopper       | Lowest value full build                             | Negative  | at least 2 teammates must have a full build |
 | ✅  |      | Underpants Gnome     | Highest kill participation                          | Positive  |
-| ⬜  |      | First Blood King     | First blood                                         | Positive  | too simple, maybe first blood + first structure? |
+| ✅  |      | Trailblazer          | First blood + first brick                           | Positive  | First blood was too simple                  |
```

Comment:
Maybe "structure" would be more clear instead of "brick"?

Reply:
Brick is the equivalent of blood.
I think it should be either "First champion kill + first structure destroyed" or "First blood + first brick"